tests/e2e/specs/in-game/reconnecting.spec.js

```diff
@@ -699,3 +699,331 @@ describe('Reauthenticating in game', () => {
+    cy.get('[data-opponent-point-card=7-3]').should('be.visible').click();
+    cy.get('[data-cy=game-over-dialog]').should('be.visible');
+    cy.reload();
+    cy.get('[data-cy=game-over-dialog]').should('be.visible');
+  });
+
+  it('Dialogs persist after refreshing when game is over by playing points from a seven', () => {
+    cy.loadGameFixture(0, {
+      p0Hand: [Card.SIX_OF_DIAMONDS, Card.SEVEN_OF_SPADES],
+      p0Points: [Card.SEVEN_OF_HEARTS, Card.SEVEN_OF_DIAMONDS],
+      p0FaceCards: [],
+      p1Hand: [],
+      p1Points: [],
+      p1FaceCards: [],
+      topCard: Card.SEVEN_OF_SPADES,
+      deck:[]
+    });
+
+    cy.get('[data-player-hand-card=7-3]').click();
+    cy.get('[data-move-choice=oneOff]').should('be.visible').click();
+    cy.resolveOpponent();
+    cy.get('[data-top-card=7-3]').click();
+    cy.get('[data-move-choice=points]').should('be.visible').click();
+    cy.get('[data-cy=game-over-dialog]').should('be.visible');
+    cy.reload();
+    cy.get('[data-cy=game-over-dialog]').should('be.visible');
+  });
+
+  it('Brings player to the rematch game when player hits rematch, disconnects, then refreshes after opponent hit rematch and started game', () => {
+    cy.concedeOpponent();
+
+    cy.get('[data-cy=gameover-rematch')
+      .should('not.be.disabled')
+      .click();
+
+    cy.get('[data-cy=my-rematch-indicator]')
+      .find('[data-cy="lobby-card-container"]')
+        .should('have.class', 'ready');
+
+    cy.window()
+      .its('cuttle.authStore')
+      .then((store) => store.disconnectSocket());
+
+    cy.get('[data-cy=opponent-rematch-indicator]')
+      .find('[data-cy="lobby-card-container"]')
+        .should('not.have.class', 'ready');
+
+    cy.url().then((url) => {
+      const oldGameId = Number(url.split('/').pop());
+      cy.wrap(oldGameId).as('oldGameId');
+      cy.rematchAndJoinRematchOpponent({ gameId: oldGameId });
+    });
+
+    cy.get('[data-cy=opponent-rematch-indicator]')
+      .find('[data-cy="lobby-card-container"]')
+        .should('not.have.class', 'ready');
+
+    cy.reload();
+
+    cy.get('#game-over-dialog')
+      .should('not.exist');
+    // Player should now be in new game
+    cy.get('[data-player-hand-card]').should('have.length', 6);
+
+    cy.url().then((url) => {
+      cy.get('@oldGameId').then((oldGameId) => {
+        const currentGameId = Number(url.split('/').pop());
+        const expectedGameId = oldGameId + 1;
+
+        expect(currentGameId).to.eq(expectedGameId, 'Expected current url to point to new game id, but it did not');
+      });
+    });
+
+    cy.get('[data-opponent-hand-card]').should('have.length', 5);
+    cy.get('#turn-indicator').should('contain', "OPPONENT'S TURN");
+    cy.drawCardOpponent();
+    cy.get('[data-opponent-hand-card]').should('have.length', 6);
+  });
+
+  it('Brings player to the rematch game when player hits rematch, disconnects, then reconnects socket after opponent hit rematch and started game', function () {
```

Comment:
For consistency, let's use arrow functions unless we're specifically using `this` to grab an alias (which I tried here and couldn't get to work, then forgot to roll back)
```suggestion
  it('Brings player to the rematch game when player hits rematch, disconnects, then reconnects socket after opponent hit rematch and started game', () => {
```